ztp/kube-compare-reference/compare.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+function read_dir() {
+local dir=$1
+local file
+
+for file in $(ls "$dir"); do
+    if [ -d "$dir""/""$file" ]
+    then
+      read_dir "$dir""/""$file"
+    else
+      echo "$dir""/""$file"
+    fi
+done
+}
+
+
+read_dir renderedv1 |grep yaml  > rendered_file
+read_dir ../source-crs |grep yaml  > source_file
+
+
+while IFS= read -r file1; do
+    while IFS= read -r file2; do
+        if [ "${file1##*/}" = "${file2##*/}" ]
+        then
+            diff -u "$file1" "$file2"
+            printf "\n"
+            echo "$file1" >> same_file
+        fi
+     done < rendered_file
+done < source_file
+
+
+while IFS= read -r file; do
+    sed -i "/${file##*/}/d" source_file
+done < same_file
```

Comment:
This will catch whether there are source-crs that have no equivalency in the rendered output.  Do we also need to check for the opposite condition?  Rendered CRs that do not have any matching entry in source-crs?

Reply:
Yes, I will add more steps here to check that, and output the result.